client/properties.lua

```diff
@@ -220,15 +220,14 @@ local function modifyProperty(propertyData)
     }, function(_, _, args)
         if not args then return end
         if args.action == "name" then
-            local propertyString = string.split(propertyData.name, ' ')
-            local propertyNumber = tonumber(propertyString[1])
-            local input = lib.inputDialog(Lang:t('modify_property_menu.title'), {
-                {type = 'input', label = Lang:t('modify_property_menu.name', {name = propertyData.name}), default = table.concat(propertyString, ' ', 2), required = true},
+            local propertyNumber, propertyString = string.strsplit(' ', propertyData.name, 2)
+            local input = lib.inputDialog(locale('modify_property_menu.title'), {
+                {type = 'input', label = locale('modify_property_menu.name', propertyData.name), default = propertyString, required = true},
```

Comment:
Do the ox locale part in another PR. In this one you didn't change locales to use json files and also didn't include anything in the fxmanifest.